composer.json

```diff
@@ -65,19 +18,21 @@
     "minimum-stability": "dev",
     "prefer-stable": true,
     "require": {
+        "php": "^7.4",
+        "ext-pdo": "*",
+        "psr/log": "^1.1",
         "yiisoft/arrays": "^3.0@dev",
         "yiisoft/strings": "^3.0@dev",
-        "ext-pdo": "*",
-        "php": "^7.4"
+        "yiisoft/profiler": "^3.0@dev"
     },
     "require-dev": {
-        "yiisoft/yii-console": "^3.0@dev",
-        "yiisoft/active-record": "^3.0@dev",
-        "yiisoft/db-mysql": "^3.0@dev",
-        "yiisoft/db-sqlite": "^3.0@dev",
-        "yiisoft/cache": "^3.0@dev",
         "yiisoft/di": "^3.0@dev",
         "yiisoft/log": "^3.0@dev",
+        "yiisoft/log-target-file": "^3.0@dev",
+        "yiisoft/aliases": "^3.0@dev",
+        "yiisoft/cache": "^3.0@dev",
+        "yiisoft/db-mysql": "^3.0@dev",
```

Comment:
This is still bad. There should not be anything about MySQL in the general DB package.

Reply:
https://github.com/yiisoft/db/blob/0b02674a92fb7749a98928b3eb5953d0edeb5814/src/Connection.php#L317 I can not remove the dependency because the implementation of Schema is in Mysql for Mysql, Schema is an abstract class, in any case they should not have separated Db and Db-Mysql, since it will never be independent.